### DISCUSSION TITLE
German support in JavaScript

### DISCUSSION THREAD

**Discussion by thorty** [microsoft/Recognizers-Text#2698] [General]
Hey,

is there a reason for not supporting `germany` in JavaScript?

Eg.:
`var model = recognizer.getNumberModel( Culture.German );` 
is not working and I do not see anything with german into the srccode:

![grafik](https://user-images.githubusercontent.com/3669417/130456545-422d465b-71bb-4e32-94e8-d178bb18ec2d.png)

Thanks for answers

**Comment by tellarin** [comment]
@thorty, JavaScript support for new languages propagates from .NET on a case by case basis. There is currently no plan to expose German.
Would you be interested in helping with the port?

**Comment by tellarin** [comment]
Same as #1516.
